lib/lrama/lexer.rb

```diff
@@ -7,8 +7,10 @@ class Lexer
     include Lrama::Report::Duration
 
     # s_value is semantic value
-    Token = Struct.new(:type, :s_value, keyword_init: true) do
-      Type = Struct.new(:id, :name, keyword_init: true)
+    Token = _ = Struct.new(:type, :s_value, keyword_init: true) do
```

Comment:
[note] Inserting `_` between a constant and `Struct.new` is a common workaround when we use `Struct` to define new class.